src/components/CorporateCampaign/CampaignStatus.vue

```diff
@@ -32,13 +32,14 @@
 							v-if="promoName && (promoAmount !== '$0.00') && activeCreditType !== 'lending_reward'"
 							@click="handlePromoLinkClick"
 							:class="{ 'tw-underline': inContext, 'tw-cursor-pointer': inContext }"
+							:style="inContext ? 'text-decoration: underline; cursor: pointer;' : ''"
```

Comment:
tailwind isn't even active here 🤦  yet. We can just remove this line in the KC refresh branch.